libs/langchain/langchain/document_loaders/airbyte_cdk.py

```diff
@@ -0,0 +1,27 @@
+"""Loads local airbyte json files."""
+import json
+from typing import Any, Iterator, List, Mapping, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+from airbyte_protocol.models.airbyte_protocol import AirbyteRecordMessage, AirbyteStateMessage
+from airbyte_cdk.sources.embedded.base_integration import BaseEmbeddedIntegration
+from airbyte_cdk.sources.embedded.runner import CDKRunner
```

Comment:
Where are the `airbyte_protocol` and `airbyte_cdk` dependencies defined?

Do we expect users to install them separately?

Reply:
One is a transitive dependency of the other, but airbyte-cdk is actually re-exporting the types, so we can work with just airbyte_protocol here.